### PR TITLE
Updating Editor Config for tab indent

### PR DESCRIPTION
**Pull request by chris31389**
Aligning to MS Standards for editor config

### CODE REVIEW

**Review by alexanderwjrussell — APPROVED** [review]
Looks good to me!

File: src/func-aeh-listener/src/.editorconfig [hunk 1]
```diff
@@ -10,6 +10,16 @@ indent_size = 4
...
 
+############################### 
+# MS Coding Conventions       # 
+############################### 
+[*.{csproj,xml}]
+indent_style = space 
+end_of_line = lf
+indent_size = 2 
+insert_final_newline = true 
+charset = utf-8-bom
```
Comment (ElvenSpellmaker) [inline src/func-aeh-listener/src/.editorconfig:21]: Do we have to use bom?

Comment (chris31389) [reply src/func-aeh-listener/src/.editorconfig:21]: its whats been used before, what would you suggest we change to?

Comment (ElvenSpellmaker) [reply src/func-aeh-listener/src/.editorconfig:21]: UTF 8 without the bom unless it's an MS prescribed thing, but minor, can discuss later if you'd prefer

Comment (chris31389) [reply src/func-aeh-listener/src/.editorconfig]: Updated 👍

Comment (ElvenSpellmaker) [reply src/func-aeh-listener/src/.editorconfig]: Cool nice one!

**Pull request merged**